package.json

```diff
@@ -63,6 +63,7 @@
     "create-react-context": "^0.2.1",
     "popper.js": "^1.14.1",
     "prop-types": "^15.6.1",
+    "react-lifecycles-compat": "^3.0.4",
```

Comment:
I could leave this out and use the old lifecycle methods but someone will show up and complain eventually

Reply:
a hotfix PR is not the place to add a dependency, unless it's critical to solving the bug

Reply:
The other option is to write throwaway code. It's not a hotfix the current master isn't released, if this was part of  my last PR I'd have included this